gnd/src/androidTest/java/com/google/android/gnd/TestGndApplication.java

```diff
@@ -0,0 +1,16 @@
+package com.google.android.gnd;
+
+import com.google.android.gnd.inject.DaggerTestGndApplicationComponent;
+import dagger.android.AndroidInjector;
+import dagger.android.support.DaggerApplication;
+
+public class TestGndApplication extends GndApplication {
```

Comment:
Just a reminder/placeholder to add JavaDoc to public classes here and throughout once the design settles.

Reply:
Thanks for the reminder. Added comment.